xterm/xterm-lib.pl

```diff
@@ -4,7 +4,7 @@
 use WebminCore;
 &init_config();
 our %access = &get_module_acl();
-do "$module_root_directory/websockets-lib-funcs.pl";
+do "$root_directory/websockets-lib-funcs.pl";
```

Comment:
Is websockets-libs-funcs going to be moved up to the Webmin root dir?

Reply:
Yes.